packages/angular_devkit/build_angular/BUILD.bazel

```
@@ -198,10 +198,10 @@ ts_project(
         "//:node_modules/@types/watchpack",
         "//:node_modules/esbuild",
         "//:node_modules/esbuild-wasm",
-        "//:node_modules/fast-glob",
         "//:node_modules/karma",
         "//:node_modules/karma-source-map-support",
         "//:node_modules/semver",
+        ":node_modules/tinyglobby",
```

```suggestion
        "//:node_modules/tinyglobby",
```
Bazel paths always start with `//`, only local targets can be accessed with `:`.